docs/en/02_Developer_Guides/08_Performance/02_HTTP_Cache_Headers.md

```diff
@@ -1,7 +1,188 @@
+
+Simple way to set cache control header to a non-cacheable state.
+Use this method over `privateCache()` if you are unsure about caching details.
+Takes precendence over unforced `enableCache()`, `privateCache()` or `publicCache()` calls.
+
+Removes all state and replaces it with `no-cache, no-store, must-revalidate`. Although `no-store` is sufficient
+the others are added under [recommendation from Mozilla](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control#Examples)
+
+Does not set `private` directive, use `privateCache()` if this is explicitly required
+([details](https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/http-caching#public_vs_private))
+
+### enableCache()
+
+Simple way to set cache control header to a cacheable state.
+Use this method over `publicCache()` if you are unsure about caching details.
+
+Removes `no-store` and `no-cache` directives; other directives will remain in place.
+Use alongside `setMaxAge()` to indicate caching.
+
+Does not set `public` directive. Usually, `setMaxAge()` is sufficient. Use `publicCache()` if this is explicitly required
+([details](https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/http-caching#public_vs_private))
+
+### privateCache()
+
+Advanced way to set cache control header to a non-cacheable state.
+Indicates that the response is intended for a single user and must not be stored by a shared cache.
+A private cache (e.g. Web Browser) may store the response. Also removes `public` as this is a contradictory directive.
+
+### publicCache()
+
+Advanced way to set cache control header to a cacheable state.
+Indicates that the response may be cached by any cache. (eg: CDNs, Proxies, Web browsers)
+Also removes `private` as this is a contradictory directive
+
+### Priority
+    
+Each of these highlevel methods has a boolean `$force` parameter which determines
+their application priority regardless of execution order.
+The priority order is as followed, sorted in descending order
+(earlier items will overrule later items): 
+
+ * `disableCache($force=true)`
+ * `privateCache($force=true)`
+ * `publicCache($force=true)`
+ * `enableCache($force=true)`
+ * `disableCache()`
+ * `privateCache()`
+ * `publicCache()`
+ * `enableCache()`
+
+## Cache Control Examples
+
+### Global opt-in for page content 
+
+Enable caching for all page content (through `Page_Controller`).
+
+```php
+<?php
+
+use SilverStripe\Control\Middleware\HTTPCacheControlMiddleware;
+use SilverStripe\CMS\Controllers\ContentController;
+
+class PageController extends ContentController
+{
+    public function init()
+    {
+        HTTPCacheControlMiddleware::singleton()
+           ->enableCache()
+           ->setMaxAge(60); // 1 minute
+        
+        parent::init();
+    }
+}
+```
+
+Note: SilverStripe will still override this preference when a session is active,
+a [CSRF token](/developer_guides/forms/form_security) token is present,
```

Comment:
I've mentioned the first two cases (validator and tokens) consistently in both docs now. Didn't mention the GET vs. POST case because I'm not sure that something worth retaining. Forms can contain sensitive, user specific data regardless of GET or POST.